pygpudrive/env/env_torch.py

```diff
@@ -429,15 +441,15 @@ def get_expert_actions(self):
         )
 
         if self.config.dynamics_model == "delta_local":
-            inferred_actions = log_trajectory.inferred_actions[:, :3]
-            inferred_actions[..., 0] = torch.clamp(
-                inferred_actions[..., 0], -6, 6
+            inferred_actions = log_trajectory.inferred_actions[:, :, :, :3]
+            inferred_actions[:, :, :, 0] = torch.clamp(
+                inferred_actions[:, :, :, 0], -6, 6
             )
-            inferred_actions[..., 1] = torch.clamp(
-                inferred_actions[..., 1], -6, 6
+            inferred_actions[:, :, :, 1] = torch.clamp(
+                inferred_actions[:, :, :, 1], -6, 6
             )
-            inferred_actions[..., 2] = torch.clamp(
-                inferred_actions[..., 2], -torch.pi, torch.pi
+            inferred_actions[:, :, :, 2] = torch.clamp(
+                inferred_actions[:, :, :, 2], -torch.pi, torch.pi
```

Comment:
This syntax seems worse than the previous syntax?

Reply:
My reasoning was that this makes it explicit which axis is indexed. Happy to revert these changes, however

Reply:
It still is explicit, it's the last axis.